src/components/switch/index.tsx

```diff
@@ -121,8 +123,8 @@ class Switch extends BaseComponent {
   }
 
   getSwitchStyle() {
-    const {value, onColor, offColor, style: propsStyle, disabled, disabledColor} = this.getThemeProps();
-    const style = [this.styles.switch];
+    const {value, onColor, offColor, style: propsStyle, disabled, disabledColor} = this.props;
+    let style: any = [this.styles.switch];
```

Comment:
1. Why change from `const` to `let`?
2. Don't use `any`, unless you have to.

Reply:
I tried something with this let and forgot to return it to const :)